spec/metka/post_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Metka::Post, :model do
+  let!(:tag1)       { 'tag1' }
+  let!(:tag2)       { 'tag2' }
+  let!(:material1)  { 'material1' }
+  let!(:material2)  { 'material2' }
```

Comment:
Since we have the Post model, can categories be used instead of materials?

Reply:
Updated